### PR TITLE
【Hackathon 6th Article No.6】稀疏计算的使用指南

### PR DESCRIPTION
**Pull request by AndSonder**

### CODE REVIEW

**Comment by AndSonder** [comment]
@zhwesky2010 请问研发老师可以帮忙 review 一下嘛

**Comment by AndSonder** [comment]
@zhwesky2010 请问研发老师还有时间帮忙 review 吗，15 号是不是就截至了 😭

File: rfcs/Article/20240412_guide_to_using_sparse.md [hunk 1]
```diff
@@ -0,0 +1,372 @@
+# Paddle 稀疏计算 使用指南
+
+## 1. 稀疏格式介绍
+
+### 1.1 稀疏格式介绍
+
+稀疏矩阵是一种特殊的矩阵,其中绝大多数元素为0。与密集矩阵相比,稀疏矩阵可以节省大量存储空间,并提高计算效率。
```
Comment (zhwesky2010) [inline rfcs/Article/20240412_guide_to_using_sparse.md:7]: 统一下中文标点符号 `，`

File: rfcs/Article/20240412_guide_to_using_sparse.md [hunk 2]
```diff
@@ -0,0 +1,372 @@
...
+Epoch 6, Loss: 14.831659317016602
+Epoch 7, Loss: 14.826210975646973
+Epoch 8, Loss: 14.820586204528809
+Epoch 9, Loss: 14.81474781036377
+Epoch 10, Loss: 14.808649063110352
+```
+
+### 4.2 图神经网络
+
+基于图神经网络的稀疏计算是一种处理非欧几里得结构化数据的有效方法。在这种情况下,数据可以表示为一个图,其中节点表示实体,边缘表示它们之间的关系。图神经网络能够学习节点表示,同时捕获图结构的拓扑信息和节点属性。
```
Comment (zhwesky2010) [inline rfcs/Article/20240412_guide_to_using_sparse.md:259]: a.py改一个更规范的名字，然后以链接的形式加到这个里面来

File: rfcs/Article/a.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+        h = self.relu(h)
+        h = self.gcn2(h, adj)
+        return h
+
+
+# 创建模型
+model = GCN()
+
+# 前向传播
+logits = model(node_feat, adj)
```
Comment (zhwesky2010) [inline rfcs/Article/a.py:57]: 有两张图片无法预览，你这边能看到吗

infoflow 2024-05-13 11-45-42

Comment (AndSonder) [reply rfcs/Article/a.py:57]: > 有两张图片无法预览，你这边能看到吗
> 
> infoflow 2024-05-13 11-45-42

可以看到，

File: rfcs/Article/20240412_guide_to_using_sparse.md (hunk 2 above)
Comment (AndSonder) [reply rfcs/Article/20240412_guide_to_using_sparse.md:259]: > a.py改一个更规范的名字，然后以链接的形式加到这个里面来

这个 a.py 是忘了删掉了，我删掉一下，代码已经都在文章里面写了

File: rfcs/Article/20240412_guide_to_using_sparse.md (hunk 1 above)
Comment (AndSonder) [reply rfcs/Article/20240412_guide_to_using_sparse.md]: > 统一下中文标点符号 `，`

done

**Comment by AndSonder** [comment]
@zhwesky2010 请问还有其他review意见嘛

**Review by zhwesky2010 — APPROVED** [review]
LGTM
